DOC-2181: add improvement documentation for TINY-10348 & TINY-10349 to the 6.8 release notes. (#2999)
tinymce/tinymce-docs: modules/ROOT/partials/security/securing-embedded-external-resources.adoc

[[sandbox-iframes-option]]
==== `sandbox_iframes` option

> This option controls whether iframe elements will be updated with the `sandbox=""` attribute to mitigate malicious intent from being inserted into the editor.

Reviewer: ```suggestion
This option controls whether the editor will add a `sandbox=""` attribute to all `<iframe>` elements. This will restrict the iframe’s embedded resource from performing potentially malicious actions including scripting, file downloads, browser popups, passing the same-origin policy, among others. Reference: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#sandbox[MDN].
```

*Type:* `+Boolean+`

*Possible values:* `true`, `false`

*Default value:* `false`

===== Example: using `sandbox_iframes` option

[source,js]
----
tinymce.init({
  selector: 'textarea',  // change this value according to your html

>   sandbox_iframes: 'false' // default

Reviewer: ```suggestion
  sandbox_iframes: false // default
```

});
----

[[convert-unsafe-embeds]]
==== `convert_unsafe_embeds` option

> This option controls whether an `<object>` and `<embed>` elements will be converted to more restrictive alternatives, namely `<img>` for image MIME types, `<video>` for video MIME types, `<audio>` audio MIME types, or `<iframe>` for other or unspecified MIME types.

Reviewer: ```suggestion
This option controls whether an `<object>` and `<embed>` elements will be converted to more restrictive alternatives, namely `<img>` for image MIME types, `<video>` for video MIME types, `<audio>` audio MIME types, or `<iframe>` for other or unspecified MIME types. 

When converted to `<img>`, `<video>`, or `<audio>`, this prevents the embedded resource from performing potentially malicious actions including scripting, file downloads, browser popups, passing the same-origin policy, among others. Enable the `sandbox_iframes` option in addition to ensure <iframe> conversions are also neutralised.
```

*Type:* `+Boolean+`

*Possible values:* `true`, `false`

*Default value:* `false`

===== Example: using `convert_unsafe_embeds` option

[source,js]
----
tinymce.init({
  selector: 'textarea',  // change this value according to your html

>   convert_unsafe_embeds: 'false' // default

Reviewer: ```suggestion
  convert_unsafe_embeds: false // default
```
